src/core/include/openvino/op/mvn.hpp

```diff
@@ -116,6 +116,10 @@ class OPENVINO_API MVN : public Op {
 
     std::shared_ptr<Node> clone_with_new_inputs(const OutputVector& new_args) const override;
 
+    bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
+
+    bool has_evaluate() const override;
```

Comment:
Do we have customer request for this evaluate method?
What is about opset2::MVN?

Reply:
Yes, we have customer request for this evaluate method, only for last MVN version.